src/libImaging/QuantOctree.c

```diff
@@ -149,10 +149,17 @@ count_used_color_buckets(const ColorCube cube) {
 static void
 avg_color_from_color_bucket(const ColorBucket bucket, Pixel *dst) {
    float count = bucket->count;
-   dst->c.r = (int)(bucket->r / count);
-   dst->c.g = (int)(bucket->g / count);
-   dst->c.b = (int)(bucket->b / count);
-   dst->c.a = (int)(bucket->a / count);
+   if (bucket->count != 0) {
```

Comment:
Minor: I suggest that this line would be better as `if (count != 0) {` - that is variable that we are concerned with in the next block.